src/openrct2-ui/UiContext.Linux.cpp

```diff
@@ -364,6 +364,8 @@ namespace OpenRCT2 { namespace Ui
 
         static void ThrowMissingDialogApp()
         {
+            IUiContext * uiContext = GetContext()->GetUiContext();
+            uiContext->ShowMessageBox("Neither KDialog nor Zenity are installed. Please install one.");
```

Comment:
Would this message be localisable, or it is too early in the process?

Reply:
I have no idea, but it is run very early, and the message directly preceeding it also not translated. Based on that, I'd wager a solid Probably not, but Maybe.

Reply:
Nope. You would need to reorganise the code below so that language is loaded before all this code.

https://github.com/OpenRCT2/OpenRCT2/blob/2d993671599ffd133b56694f68de1837ff605cbb/src/openrct2/Context.cpp#L296-L327

Reply:
Well, I'd like to also have the 'please select your rct2 dir' message translated, so that's something I'd like to reorganise.

Kenton: if you feel comfortable changing this: please do so. Otherwise, I can do it.